### PR TITLE
[PUBDEV-5126] Ensure that correct version is produced for both sdist …

### PR DESCRIPTION
**Pull request by jakubhava**
…and whl, clean up a little bit

At the end I tried to make this change as small as possible to minimise the side effects.

We use version files to obtain versions which mean that we use the same version for different build tools (python dist tool independent). 

This is also a good step for automating conda. @abal5 that is next step

### CODE REVIEW

**Comment by jakubhava** [comment]
@h2o-ops please test!

**Comment by jakubhava** [comment]
THis still need small fix because currently the version file is not parted of the distribution

**Comment by jakubhava** [comment]
@h2o-ops please test!

File: h2o-py/setup.py [hunk 1]
```diff
@@ -3,14 +3,18 @@
...
 
 here = os.path.abspath(os.path.dirname(__file__))
 
 # Get the long description from the relevant file
 with open(os.path.join(here, 'DESCRIPTION.rst'), encoding='utf-8') as f:
     long_description = f.read()
 
+# Get the version from the relevant file
+with open(os.path.join(here, 'h2o/version.txt'), encoding='utf-8') as f:
+    version = f.read()
```
Comment (mmalohlava) [inline h2o-py/setup.py]: Can we have default here, which will generate crippled version?

Comment (jakubhava) [reply h2o-py/setup.py]: We can, but why do you think we need it ?

The version file is part of distribution ( ie will be always there )

**Pull request merged**